pkg/helpers/image/test/util.go

```diff
@@ -287,6 +289,79 @@ func DS(namespace, name string, containerImages ...string) kappsv1.DaemonSet {
 	}
 }
 
+// CronJobList turns the given stateful sets into JobList.
+func CronJobList(cjs ...batchv1beta1.CronJob) batchv1beta1.CronJobList {
+	return batchv1beta1.CronJobList{
+		Items: cjs,
+	}
+}
+
+// CronJob creates and returns a CronJob object.
+func CronJob(namespace, name string, containerImages ...string) batchv1beta1.CronJob {
+	return batchv1beta1.CronJob{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+			Name:      name,
+			SelfLink:  "/apis/apps/v1/deployments/" + name,
```

Comment:
I know it's tests, but let's make it still accurate :wink:

Reply:
Addressed.